features/step_definitions/basics_steps.rb

```diff
@@ -0,0 +1,20 @@
+When("I visit the {string}") do |string|
```

Comment:
If we change "landing page" to `landing page`, maybe we can change this step to the same as Zane and Carla, i.e without any {string} and |string| arguments?

Reply:
Changed!